src/main/java/mate.academy/Main.java

```diff
@@ -3,14 +3,15 @@
 import java.util.List;
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
-import mate.academy.service.ProductService;
+import mate.academy.service.impl.ProductServiceImpl;
 
 public class Main {
 
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductServiceImpl productService = (ProductServiceImpl) injector
+                .getInstance(ProductServiceImpl.class);
```

Comment:
You should not cast the result of `injector.getInstance()` to `ProductServiceImpl`. Instead, use the interface type `ProductService`. This ensures that you are programming to an interface, not an implementation, which is a good practice in Java.